themes/rjorgenson/rjorgenson.theme.sh

```diff
@@ -14,88 +14,82 @@ SCM_THEME_PROMPT_CLEAN=" ${bold_green}✓${normal}"
 SCM_GIT_CHAR="${STRING_COLOR}±${normal}"
 SCM_SVN_CHAR="${bold_cyan}⑆${normal}"
 SCM_HG_CHAR="${bold_red}☿${normal}"
+PROMP_CHAR="${BRACKET}➞ ${normal}"
 
 #Mysql Prompt
 export MYSQL_PS1="(\u@\h) [\d]> "
 
+TITLEBAR=""
 case $TERM in
-        xterm*)
-        TITLEBAR="\[\033]0;\w\007\]"
-        ;;
-        *)
-        TITLEBAR=""
-        ;;
+        xterm*) TITLEBAR="\[\033]0;\w\007\]" ;;
 esac
 
-PS3=">> "
+if [[ $OSTYPE =~ "darwin" ]]; then
+    echo "${BRACKET}➞  ${normal}"
+fi
 
 __my_rvm_ruby_version() {
-    local gemset=$(echo $GEM_HOME | awk -F'@' '{print $2}')
-  [ "$gemset" != "" ] && gemset="@$gemset"
-    local version=$(echo $MY_RUBY_HOME | awk -F'-' '{print $2}')
-    local full="$version$gemset"
-  [ "$full" != "" ] && echo "${BRACKET_COLOR}[${STRING_COLOR}$full${BRACKET_COLOR}]${normal}"
+    local gemset ; gemset=$(echo "${GEM_HOME}" | awk -F'@' '{print $2}')
```

Comment:
I'm interested in why you have separated the declaration and the assignments here as `local gemset ; gemset=...`.

Reply:
Shellcheck SC2155: Declare and assign separately to avoid masking return values.

You don't need to always separate the declaration and assignment, just when assigning results of subshells or external commands.

Reply:
Thanks, but in this case, we do not use the exit status, and also we will never turn on `set -e` in the interactive shells. I must say this is some false-positive warning by shellcheck. But maybe we should do it to work around shellcheck.

Reply:
This is also acceptable as the workaround for shellcheck.